src/common/WorkQueue.h

```diff
@@ -434,7 +434,7 @@ class ThreadPool : public md_config_obs_t {
   };
 private:
   vector<WorkQueue_*> work_queues;
-  int last_work_queue;
+  int next_work_queue;
```

Comment:
er... Since all other vars are initialized in initializer list yet, I think it may makes the reader weird: only this variable is initialized in its declare place. If you prefer this style, I suggest we create a new pr to clean this.